inheritance/src/main/java/inheritance/Review.java

```diff
@@ -1,14 +1,14 @@
 package inheritance;
 
-
-
 public class Review {
     //Instance variables
     private String author;
     private String textBody;
     private int star;
+    private String movie;
     private Restaurant restaurant;
     private Shop shop;
+//    private Theater theater;
```

Comment:
If a `Review` can store a `Restaurant` or `Shop` then shouldn't it also store a `Theater`? It's good to strive for consistency in your code so other developers know what to expect.